File: app/src/main/java/knf/kuma/tv/search/TVSearch.java
package knf.kuma.tv.search;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

import knf.kuma.tv.TVBaseActivity;

public class TVSearch extends TVBaseActivity {
    public static void start(Context context) {
        context.startActivity(new Intent(context, TVSearch.class));
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addFragment(new TVSearchFragment());
    }
}

File: app/src/main/java/knf/kuma/tv/search/TVSearch.kt
package knf.kuma.tv.search

import android.content.Context
import android.content.Intent
import android.os.Bundle
import knf.kuma.tv.TVBaseActivity
import kotlinx.coroutines.ExperimentalCoroutinesApi
import kotlin.contracts.ExperimentalContracts


@ExperimentalCoroutinesApi
@ExperimentalContracts
class TVSearch : TVBaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addFragment(TVSearchFragment())
    }

    companion object {
        fun start(context: Context) {
            context.startActivity(Intent(context, TVSearch::class.java))
        }
    }
}
